app/models/installed_heat_source.rb

```diff
@@ -0,0 +1,18 @@
+class InstalledHeatSource
+  include Virtus.model
+
+  attribute :key, String
+  attribute :name, String
+  attribute :units, Float
+  attribute :heat_production, Float
```

Comment:
> Regard production though, wouldn't calling it demand make it easier to get a demand-scaled network curve?

How do you mean? This might seem like a stupid question: but Is the `demand` of a heat technology equal to the `production`?